### PR TITLE
Add user functionality mgt error codes

### PR DESCRIPTION
**Pull request by piraveena**

### CODE REVIEW

File: en/docs/references/error-catalog.md [hunk 1]
```diff
@@ -1525,3 +1525,67 @@ This document describes all the REST API error codes that are used in WSO2 Ident
   </tbody>
 </table>
 </div>
+
+
+## Uder Functionality Management Errors
```
Comment (sherenewso2) [inline en/docs/references/error-catalog.md:1530]:
```suggestion
## User Functionality Management Errors
```

File: en/docs/references/error-catalog.md [hunk 2]
```diff
@@ -1525,3 +1525,67 @@ This document describes all the REST API error codes that are used in WSO2 Ident
...
+        <td>UFM-10002</a></td>
+        <td>403</td>
+        <td>User is not permitted to unlock.</td>
+        <td>The user is not permitted to unlock the functionality.</td>
+      </tr>
+      <tr>
+        <td>UFM-10003</a></td>
+        <td>404</td>
+        <td>Invalid UserID provided.</td>
+        <td>The provided userId is invalid.</td>
```
Comment (sherenewso2) [inline en/docs/references/error-catalog.md:1557]:
```suggestion
        <td>The provided UserID is invalid.</td>
```

File: en/docs/references/error-catalog.md [hunk 3]
```diff
@@ -1525,3 +1525,67 @@ This document describes all the REST API error codes that are used in WSO2 Ident
...
+      </tr>
+      <tr>
+        <td>UFM-10004</a></td>
+        <td>400</td>
+        <td>Invalid Request.</td>
+        <td>The request provided is invalid.</td>
```
Comment (sherenewso2) [inline en/docs/references/error-catalog.md:1563]:
```suggestion
        <td>The provided request is invalid.</td>
```

File: en/docs/references/error-catalog.md [hunk 4]
```diff
@@ -1525,3 +1525,67 @@ This document describes all the REST API error codes that are used in WSO2 Ident
...
+        <td>UFM-15002</a></td>
+        <td>500</td>
+        <td>Failed to lock the functionality.</td>
+        <td>A system error occurred while locking the functionality.</td>
+      </tr>
+      <tr>
+        <td>UFM-15003</a></td>
+        <td>500</td>
+        <td>Failed to unlock the functionality.</td>
+        <td>A system error occurred while locking the functionality.</td>
```
Comment (sherenewso2) [inline en/docs/references/error-catalog.md:1587]:
```suggestion
        <td>A system error occurred while unlocking the functionality.</td>
```

**Comment by piraveena** [comment]
move to https://github.com/wso2/docs-is/pull/2271

**Pull request closed without merging**